packages/react/src/checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,94 @@
+import * as RadixCheckbox from "@radix-ui/react-checkbox";
+import * as RadixLabel from "@radix-ui/react-label";
+import { useId } from "@reach/auto-id";
+import { type ComponentPropsWithRef, type ReactNode, forwardRef } from "react";
+
+import type { ExtendProps } from "../utils";
+
+import { Box } from "../box";
+import { Flex } from "../flex";
+import { Text } from "../text";
+import * as styles from "./Checkbox.css";
+import { IconChecked } from "./icons-svg/IconChecked";
+import { IconIndeterminate } from "./icons-svg/IconIndeterminate";
+import { IconUnchecked } from "./icons-svg/IconUnchecked";
+
+type CheckboxProps = ExtendProps<
+  ComponentPropsWithRef<typeof Box>,
+  {
+    checked?: "indeterminate" | boolean;
+    children: ReactNode;
```

Comment:
Box already has the children prop. I think we don't need to expose the children prop again.